tests/framework/CodeInterfaceTests/Neutrino/gold/Neutrino_Base/out_dump_0.csv

```diff
@@ -0,0 +1,302 @@
+time,result
+0.0292761,0.0266502

```

Comment:
you should remove this... you are not testing against this... you just test _"Neutrino_Base/out_dump.csv"_ in your **tests** file